backend/app/services/llm/jobs.py

```
@@ -170,27 +202,66 @@ def execute_job(
             else:
                 config_blob = config.blob
 
+            user_sent_config_provider = ""
```

Ruff (0.14.14)

[error] Local variable `user_sent_config_provider` is assigned to but never used

Remove assignment to unused variable `user_sent_config_provider`

(F841)